packages/quantic/playwright.config.ts

```diff
@@ -0,0 +1,78 @@
+import {defineConfig, devices} from '@playwright/test';
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+import dotenv from 'dotenv';
+import path from 'path';
+dotenv.config({path: path.resolve(__dirname, '.env')});
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig({
+  testDir: './force-app',
+  testMatch: '*.e2e.ts',
+  /* Run tests in files in parallel */
+  fullyParallel: true,
+  /* Fail the build on CI if you accidentally left test.only in the source code. */
+  forbidOnly: !!process.env.CI,
+  /* Retry on CI only */
+  retries: process.env.CI ? 2 : 0,
+  /* Opt out of parallel tests on CI. */
+  workers: process.env.CI ? 1 : undefined,
+  /* Reporter to use. See https://playwright.dev/docs/test-reporters */
+  reporter: 'html',
```

Comment:
In Atomic we use different reporters in the CI:

```suggestion
  reporter: process.env.CI
    ? [['html'], ['list'], ['github'], ['blob']]
    : [['html'], ['list']],
```

Reply:
This is to upload the playwright test report to https://github.com/coveo/ui-kit-prs . Will you do the same for quantic ? This is really helpful for quick debugging.

Reply:
very nice! thanks for the info, we will follow the same approach 👍